metricbeat/module/kubernetes/controllermanager/_meta/test/metrics.1.32.expected

```diff
@@ -0,0 +1 @@
+null
```

Comment:
Leaving as draft for now, I need to understand why these files were not properly generated.

Reply:
@constanca-m Do you have any guidance on generating these expectation files? I was able to retrieve the metrics from a Kubernetes cluster running v1.32, but go test -data only generates files containing null. I'd appreciate it if you could point me to some relevant documentation or someone who could help. Thanks!

Reply:
I don't think I have worked on the expected files that do not come from KSM. But for KSM we have a [README](https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/_meta/test/README.md) file, and I think generating these files would be the same.

Did you run the command as `go test . --data` inside `metricbeat/module/kubernetes/controllermanager` directory (do not get inside the `_meta` directory)?

Reply:
Sorry for the late delay. This week I was able to revisit this task. I managed to generate the necessary test files for the proxy, scheduler and controllermanager metrics. I had to figure things out on my own since there is no documentation for how to fetch such metrics from an existing cluster. I took notes for every step and will be compiling a doc with instructions. Thanks!